drivers/dac/Kconfig.dacx0501

```diff
@@ -0,0 +1,22 @@
+# DAC configuration options
+
+# Copyright (c) 2023 Google, LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config DAC_DACX0501
+	bool "TI DACx0501 DAC driver"
+	default n
```

Comment:
That's superfluous.

Reply:
Did you mean to replace it with the auto generated DT_HAS_DAC_DACX0501_ENABLED ?

Reply:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults